e2e/support/helpers/e2e-metabot-helpers.ts

```diff
@@ -0,0 +1,81 @@
+import type { StaticResponse } from "cypress/types/net-stubbing";
+
+import { popover } from "./e2e-ui-elements-helpers";
+
+export function assertChatVisibility(visiblility: "visible" | "not.visible") {
+  cy.findByTestId("metabot-chat").should(
+    visiblility === "visible" ? "exist" : "not.exist",
```

Comment:
```suggestion
    visiblility === "visible" ? "be.visible" : "not.exist",
```

you want the more robust assertion of visibility